MVC/application/model/userAccount.php

```diff
@@ -62,6 +66,82 @@ public function showProfile($userEmail) {
       $userData[5]  = $row['UserImg'];
 
       return $userData;
+      $this->conn->close();
+    }
+
+    public function storePost($userEmail, $comment, $image) {
+      $verify = "SELECT UserId, FirstName FROM Account WHERE UserEmail = '$userEmail'";
+      $result = $this->conn->query($verify);
+
+      $row = $result->fetch_assoc();
+
+      if($row > 0) {
+        $connection = new mysqli($this->host, $this->user, $this->pwd, "UserPosts");
```

Comment:
Another alternative way of  doing this is we can add a class where we establish connection with the database and extend that class. So that we need not establish connection with database everytime

Reply:
Okay I will change it later.